File: ios/Runner/SimpleUtils.m
//
//  SimpleUtils.m
//  Runner
//
//  Created by Fida on 08.02.21.
//
#import "SimpleUtils.h"
 
@implementation SimpleUtils

+ (BOOL) stringToBool: (NSString *) myString {
    bool myBool = [myString isEqual: @"true"] ? true : false;
    return myBool;
}
@end

File: android/app/src/main/java/de/uni_bremen/informatik/SimpleUtils.java
package de.uni_bremen.informatik;

/**
 * Created by Fida on 11.02.21.
 */
public class SimpleUtils {

    public static float boolToFloatFlag(boolean isTrue){
        return isTrue ? 1f : 0f;
    }
    public static boolean stringToBool(String boolValue){
        return Boolean.parseBoolean(boolValue);
    }
}
